src/telegrafs/components/CollectorCard.tsx

```diff
@@ -82,31 +92,46 @@ class CollectorRow extends PureComponent<
   }
 
   private get contextMenu(): JSX.Element {
+    const settingsRef: RefObject<HTMLButtonElement> = createRef()
+
     return (
-      <Context>
-        <Context.Menu
-          icon={IconFont.Duplicate}
-          color={ComponentColor.Secondary}
-          testID="telegraf-clone-menu"
-        >
-          <Context.Item
-            label="Clone"
-            action={this.cloneTelegraf}
-            testID="telegraf-clone-button"
-          />
-        </Context.Menu>
-        <Context.Menu
-          testID="telegraf-delete-menu"
-          icon={IconFont.Trash}
-          color={ComponentColor.Danger}
-        >
-          <Context.Item
-            testID="telegraf-delete-button"
-            label="Delete"
-            action={this.handleDeleteConfig}
-          />
-        </Context.Menu>
-      </Context>
+      <FlexBox margin={ComponentSize.ExtraSmall}>
+        <ConfirmationButton
+          color={ComponentColor.Colorless}
+          icon={IconFont.Trash_New}
+          shape={ButtonShape.Square}
+          size={ComponentSize.ExtraSmall}
+          confirmationLabel="Yes, Delete this configuration"
+          onConfirm={this.handleDeleteConfig}
+          confirmationButtonText="Confirm"
+          testID="context-delete-menu"
+        ></ConfirmationButton>
```

Comment:
can this be converted into a single closing tag?

```ts
<ConfirmationButton
...
/>
```

Reply:
yup!